lib/browser/results.js

```diff
@@ -30,6 +30,7 @@ createMethods(Results.prototype, objectTypes.RESULTS, [
     'filtered',
     'sorted',
     'snapshot',
+    'subscribe',
```

Comment:
Have we reached binding-level agreement on the `subscribe` terminology? My only concern is that it doesn't seem descriptive enough and may be confused with subscribing for notifications.

Reply:
I don't think there's been any serious discussion about the specific naming used here. If you're concerned about this naming, we have very little time to resolve it, so I'd suggest bringing it up in Slack so we can hammer it out promptly.